core/commands.go

```diff
@@ -0,0 +1,297 @@
+package core
+
+type DiceCmdMeta struct {
+	Name string
+	Info string
+	Docs string
+	Eval func([]string) []byte
+}
+
+var (
+	diceCmds = map[string]DiceCmdMeta{}
+
+	pingCmdMeta = DiceCmdMeta{
+		Name: "PING",
+		Info: "ping command info here",
```

Comment:
The Eval functions already have the docs in code comments. Can we add them here for completeness?

Reply:
the docs in code comments are different than the command should return, no ?
reference - https://redis.io/docs/latest/commands/command-docs/

Reply:
We can either use the docs from Redis's website, or use the ones in our code comments. I guess they both refer to same behaviour in different language.
Whatever you chose i'd prefer having a meaningful info here rather than the placeholder.